dts/arc/arc_axs.dtsi

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019, Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "skeleton.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "snps,archs48";
+			reg = <0>;
+		};
+
+	};
+
+	intc: arcv2-intc {
+		compatible = "snps,arcv2-intc";
+		interrupt-controller;
+		#interrupt-cells = <2>;
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		interrupt-parent = <&intc>;
+		ranges;
+
+		ddr0: memory@80000000 {
+			device_type = "memory";
+			compatible = "mmio-sram";
+			reg = <0x80000000 0x20000000>;
+		};
+
+		uart2: uart@E0022000 {
+			compatible = "ns16550";
+			clock-frequency = <33333333>;
+			reg = <0xE0022000 0x1000>;
+			label = "UART_2";
+			interrupts = <19 1>;
```

Comment:
Just noticed that we have 2 values in `interrupt` property here: `<19 1>`.
Is the second one being really used in Zephyr? Doesn't seem to be the case for me. Care to check?